MoltenVK/MoltenVK/GPUObjects/MVKImage.h

```diff
@@ -472,23 +581,54 @@ class MVKImageView : public MVKVulkanAPIDeviceObject {
 				 const VkImageViewCreateInfo* pCreateInfo,
 				 const MVKConfiguration* pAltMVKConfig = nullptr);
 
-	~MVKImageView() override;
-
 protected:
+    friend MVKImageViewPlane;
+    
 	void propagateDebugName() override;
-	id<MTLTexture> newMTLTexture();
-	void initMTLTextureViewSupport();
-	void validateImageViewConfig(const VkImageViewCreateInfo* pCreateInfo);
 
     MVKImage* _image;
+    std::vector<std::unique_ptr<MVKImageViewPlane>> _planes;
```

Comment:
Try using an `MVKSmallVector` here. It's faster, and there should never be more than 3 planes anyway.

Reply:
I also updated it for planes and memoryBindings in MVKImage.